Source/WebCore/platform/mediastream/WebRTCProvider.cpp

```diff
@@ -306,4 +307,23 @@ std::optional<MediaCapabilitiesEncodingInfo> WebRTCProvider::videoEncodingCapabi
     return { };
 }
 
+void WebRTCProvider::setPortAllocatorRange(StringView range)
+{
+    auto components = range.toStringWithoutCopying().split(':');
+    if (components.size() != 2)
+        return;
+
+    auto minPort = WTF::parseInteger<int>(components[0]).value_or(0);
+    auto maxPort = WTF::parseInteger<int>(components[1]).value_or(0);
+    if (!minPort || !maxPort)
+        return;
```

Comment:
I think you can ASSERT_NOT_REACHED and LOG in these two ifs. They come from a wrongly set user preference.

Reply:
This object is not loggable.